opus/application/static_media/js/cart.js

```diff
@@ -16,7 +16,7 @@ var o_cart = {
     maxCachedObservations: 1000,    // max number of observations to store in cache, will be updated based on screen size
     lastRequestNo: 0,
     downloadInProcess: false,
-    cartCount: 0,
+    totalObsCount : undefined,
```

Comment:
I tried to group the variables that are common to o_browse and o_cart at the top of the var declaration. I think it would be nice to keep them together. Also could add a comment saying these are in common, and another comment marking the start of the ones that are not in common.